other/utils.py

```diff
@@ -340,7 +340,7 @@ def wii_ssl_wrap_socket(sock, ssl_cert, ssl_key):
     return context.wrap_socket(sock, server_side=True)
 
 
-def create_server(server_class, server_handler,
+def create_server(server_class, server_handler, binary_loader,
```

Comment:
It's currently initialized a single time in `master_server.py` and a reference to that object is passed down to the children that require it. Since it's stateful (currently in that it tracks an incrementing version number, and in the future in that we might have it on an auto refresh timer to check for updated files periodically) I'd rather do it this way.